question28/js/forgotpassword.js

```diff
@@ -19,21 +19,44 @@ mailInputArea.addEventListener('blur', (e) => {
   toggleDisabledOfSubmitButton(checkAllValidity());
 });
 
-const init = () => {
-  const inputMailData = mailInputArea.value;
+const checkData = (emailAddressValue) => {
   const registeredUserData = JSON.parse(localStorage.getItem('userData'));
+  return ((emailAddressValue === registeredUserData.user_email));
```

Comment:
Q.
Is it okay without the ()?
```suggestion
  return emailAddressValue === registeredUserData.user_email;
```

Reply:
It's true! Thanks for pointing that out! Fixed so please confirm.

fixed commit: https://github.com/csakyo/homework/pull/43/commits/a16567ab2abd5745f39951cfcc51acb526fe6763